bundles/CoreBundle/Resources/config/class_builder.yaml

```diff
@@ -0,0 +1,63 @@
+services:
+    _defaults:
+        public: true
+
+    Pimcore\DataObject\ClassBuilder\ClassBuilderInterface: '@Pimcore\DataObject\ClassBuilder\ClassBuilder'
+    Pimcore\DataObject\ClassBuilder\ClassBuilder:
+        arguments:
+            - '@Pimcore\DataObject\ClassBuilder\FieldDefinitionDocBlockBuilderInterface'
```

Comment:
This should be possible with autowiring as well, right?

Reply:
sure